libqtile/layout/xmonad.py

```diff
@@ -147,7 +147,8 @@ class MonadTall(_SimpleLayoutBase):
         Key([modkey, "shift"], "k", lazy.layout.shuffle_up()),
         Key([modkey], "i", lazy.layout.grow()),
         Key([modkey], "m", lazy.layout.shrink()),
-        Key([modkey], "n", lazy.layout.normalize()),
+        Key([modkey], "n", lazy.layout.reset()),
+        Key([modkey, "shift"], "n", lazy.layout.normalize()),
```

Comment:
Why change the previous binding? Couldn't you add reset on shift+n, rather than changing the normalize binding?

Reply:
'n' just feel more natural for this, instead of Shift + n. I consulted several other ppl abt this and they agreed with me, so thats why i did it like this. But if changed previous keybinding is out of question, how abt smth like this? 

```python
Key([modkey], "r", lazy.layout.reset()),
Key([modkey], "n", lazy.layout.normalize()),
```

Reply:
`mod + r` is already a binding in the default config so stick with your original idea.

However, this should be added to `MonadWide` as well.

Reply:
Done. Dont mind these "Merge branch [...]" commits. I had several problems pushing this and from what I tried and read that was the only solution. Hopefully I didnt effed up something. Git is confusing